universal-application-tool-0.0.1/conf/logback.xml

```diff
@@ -33,6 +33,9 @@
   <logger name="org.pac4j" level="INFO" />
   <logger name="auth" level="INFO" />
 
+  <!-- Increase the level of this one to avoid leaking PII into logs -->
```

Comment:
Can you expand on what PII as an example?

Reply:
@bion do we have a SOP around what should or shouldn't show up in logs?

Reply:
as an example, this object is logged as info whenever a successful SAML authentication happens:
```
civiform  | [info] o.p.s.c.SAML2Credentials - Constructed SAML2 credentials: SAML2Credentials{nameId=SAMLNameID{format='urn:oasis:names:tc:SAML:2.0:nameid-format:transient', nameQualifier='null', spNameQualifier='null', spProviderId='null', value='shubharajan@google.com'}, sessionIndex='29e5a068-3a39-46dd-a824-553cf35a02e9', attributes=[SAMLAttribute{friendlyName='null', name='email', nameFormat='urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified', attributeValues=[shubharajan@google.com]}, SAMLAttribute{friendlyName='null', name='first_name', nameFormat='urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified', attributeValues=[Shubha]}, SAMLAttribute{friendlyName='null', name='last_name', nameFormat='urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified', attributeValues=[Rajan]}], conditions=SAMLConditions{notBefore=2022-03-01T04:00:22Z, notOnOrAfter=2022-03-01T04:10:22Z}, issuerId='https://civiform-staging.hub.loginradius.com/', authnContexts=[urn:oasis:names:tc:SAML:2.0:ac:classes:unspecified]}
```

Reply:
so it looks like first name, last name and email are the only sensitive info there?

Reply:
@sussman for visibility